context/app/publication/kidney-cell-states-atlas.md

```diff
@@ -0,0 +1,43 @@
+---
+title: An atlas of healthy and injured cell states and niches in the human kidney
+vitessce_conf:
+  [
+    {
+      'version': '0.1.0',
+      'name': 'View #1 name goes here',
+      'layers': [],
+      'staticLayout':
+        [
+          { 'component': 'description',
+            'x': 0,
+            'y': 0,
+            'w': 12,
+            'h': 4,
+            'props': { 'description': 'View #1 goes here' }
+           },
+        ]
+    },
+    {
+      'version': '0.1.0',
+      'name': 'View #2 name goes here',
+      'layers': [],
+      'staticLayout':
+        [
+          { 'component': 'description',
+            'x': 0,
+            'y': 0,
+            'w': 12,
+            'h': 4,
+            'props': { 'description': 'View #2 goes here' }
+           },
+        ]
+    }
+  ]
+---
+[Preprint on Biorxive](https://www.biorxiv.org/content/10.1101/2021.07.28.454201v1)
```

Comment:
```suggestion
[Preprint on BioRxiv](https://www.biorxiv.org/content/10.1101/2021.07.28.454201)
```

Unfortunately, bioRxiv adds the version number of the preprint (e.g. "v1") at the end of the URL once you visit the DOI link but redirects to the latest if it is not part of the URL. I removed it here.

Reply:
Obviously a moot point if we make the other requested changes ...